pkg/api/kubegraph/analysis/deployment.go

```diff
@@ -0,0 +1,53 @@
+package analysis
+
+import (
+	"sort"
+	"strconv"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/util/deployment"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
+)
+
+// Revision returns the revision number of the input replica set
+// TODO: Use the upstream helper when it will be available
+func revision(rs *extensions.ReplicaSet) int64 {
```

Comment:
we will get rid of this with next rebase (upstream does not have this helper public)